debugAdapterProtocol.json

```diff
@@ -3361,23 +3361,24 @@
 						"Indicates that the object is an interface.",
 						"Indicates that the object is the most derived class.",
 						"Indicates that the object is virtual, that means it is a synthetic object introducedby the\nadapter for rendering purposes, e.g. an index range for large arrays.",
-						"Indicates that a data breakpoint is registered for the object."
+						"Deprecated: Indicates that a data breakpoint is registered for the object."
```

Comment:
I might suggest saying something like: "The 'hasDataBreakpoint' attribute should generally be used instead."